graphcanvas/tests/test_graph_node_hover_tool.py

```diff
@@ -0,0 +1,88 @@
+from StringIO import StringIO
+import sys
+import unittest
+
+import networkx
+
+from enable.api import BasicEvent
+from graphcanvas.graph_node_hover_tool import GraphNodeHoverTool
+from graphcanvas.graph_container import GraphContainer
+from graphcanvas.graph_node_component import GraphNodeComponent
+
+
+def simple_callback(label):
+    print label
+
+
+class TestGraphNodeHoverTool(unittest.TestCase):
+    def setUp(self):
```

Comment:
FYI you might want to have a look at https://github.com/enthought/enable/blob/master/enable/testing.py it has some utility functions to helper with testing enable components/tools